Content.Server/GameTicking/Prototypes/SolitarySpawningPrototype.cs

```diff
@@ -0,0 +1,57 @@
+using System.ComponentModel.DataAnnotations;
+using Content.Server.Maps;
+using Content.Shared.Roles;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.GameTicking.Prototypes;
+
+/// <summary>
+/// This prototype defines details for a spawn option for SolitarySpawningRule.
+/// </summary>
+[Prototype]
+public sealed class SolitarySpawningPrototype : IPrototype
+{
+    /// <inheritdoc/>
+    [IdDataField]
+    public string ID { get; private set; } = default!;
+
+    /// <summary>
+    /// The label of the button on the Join GUI
+    /// </summary>
+    [DataField]
+    public LocId Name = string.Empty;
+
+    /// <summary>
+    /// The tooltip of the button on the Join GUI
+    /// </summary>
+    [DataField]
+    public LocId Description = string.Empty;
+
+    /// <summary>
+    /// This message is sent to each player when they spawn
+    /// </summary>
+    [DataField]
+    public LocId? WelcomeLoc;
+
+    /// <summary>
+    /// The station that will be created for the player.
+    /// </summary>
+    [DataField, Required]
```

Comment:
```suggestion
    [DataField(required: true)]
```
you are using some weird system attribute that does not exist in our codebase